icu4c/source/tools/tzcode/zic.c

```diff
@@ -3,6 +3,8 @@
 ** 2006-07-17 by Arthur David Olson.
 */
 
+#define _POSIX_C_SOURCE 200112L
```

Comment:
could you add a comment about "get symlink() declared" ?

Reply:
Done.